htdocs/web_portal/components/Get_User_Principle.php

```diff
@@ -137,7 +137,7 @@ function Get_User_AuthType() {
 }
 
 /**
- * Get the user's principle string (x509 DN from certificate or from SAML attribute).
+ * Get the user's principle string (X.509 DN from certificate or from SAML attribute).
```

Comment:
I believe what we should be referring to here, and probably throughout the code, is _principal_ not _principle_, but, if correct, it's rather late to change it now.

Reply:
Tbh 🤷... In #368, I opted for `identifier`, because that's how we refer them while speaking.